app/agents/voice/automatic/tools/breeze/analytics.py

```
@@ -50,7 +50,7 @@ async def _make_breeze_request(params: FunctionCallParams, operational_tab: str)
         end_time_iso = end_time_utc.isoformat().replace('+00:00', 'Z')
 
     except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)